app/models/resource_action_workflow.rb

```diff
@@ -21,14 +21,19 @@ def initialize(values, requester, resource_action, options = {})
 
   Vmdb::Deprecation.deprecate_methods(self, :dialogs => :dialog)
 
-  def submit_request
+  def submit_request(data = {})
+    update_dialog_field_values(data) if data.present?
```

Comment:
if `data` is `{}` because it wasn't passed in, this will still evaluate to `true`, and then it will end up calling `data.values.first` down on line 34, which will then fail at https://github.com/ManageIQ/manageiq/blob/master/app/models/dialog.rb#L92 since you can't call `with_indifferent_access` on `nil`.